packages/runtime/datastore-definitions/src/dataStoreRuntime.ts

```diff
@@ -61,8 +61,6 @@ export interface IFluidDataStoreRuntime extends
 
     readonly clientId: string | undefined;
 
-    readonly documentId: string;
```

Comment:
You will have to add this to Breaking.md since we are changing public facing interface. Same for dataStoreContext.